packages/next/build/webpack-config.ts

```diff
@@ -660,7 +660,9 @@ export default async function getBaseWebpackConfig(
       nodeEnv: false,
       splitChunks: isServer ? false : splitChunksConfig,
       runtimeChunk: isServer
-        ? undefined
+        ? isWebpack5 && !isLikeServerless
```

Comment:
pretty sure this might have been what was biting me when trying to connect remote containers for SSR